
jetson:
  bags:
    zed2i:
      - svo # SVO recording, not rosbag. Mutually exclusive: won't record zed ros topics if svo is set here.
    utils:
      - /gt_box/alphasense_driver_node/imu
      - /gt_box/health_status/jetson/status
      - /gt_box/health_status/pi/status
    ap20:
      - /gt_box/ap20/imu
      - /gt_box/ap20/prism_position
    ap20_aux:
      - /ap20/tps
      - /ap20/imu
      - /gt_box/ap20/debug
    clock_jetson:
      - /clock
    adis:
      - /gt_box/adis16475_node/imu
nuc:
  bags:
    cpt7_local:
      - cpt7_local
    utils:
      - /gt_box/health_status/nuc/status
    livox:
      - /gt_box/livox/imu
      - /gt_box/livox/imu_SI
    alphasense:
      - /gt_box/alphasense_driver_node/debug_info
      - /gt_box/alphasense_driver_node/imu
    cpt7:
      - /gt_box/cpt7/bestgnsspos
      - /gt_box/cpt7/bestpos
      - /gt_box/cpt7/bestutm
      - /gt_box/cpt7/bestvel
      - /gt_box/cpt7/corrimu
      - /gt_box/cpt7/gps/fix
      - /gt_box/cpt7/gps/gps
      - /gt_box/cpt7/gps/imu
      - /gt_box/cpt7/heading2
      - /gt_box/cpt7/imu/data_raw
      - /gt_box/cpt7/insconfig
      - /gt_box/cpt7/inspva
      - /gt_box/cpt7/inspvax
      - /gt_box/cpt7/insstdev
      - /gt_box/cpt7/odom
      - /gt_box/cpt7/oem7raw
      - /gt_box/cpt7/ppppos
      - /gt_box/cpt7/rxstatus
      - /gt_box/cpt7/terrastarinfo
      - /gt_box/cpt7/terrastarstatus
      - /gt_box/cpt7/time
    tf:
      - /tf
      - /tf_static